MAVProxy/modules/mavproxy_help.py

```diff
@@ -45,13 +45,15 @@ def __init__(self, mpstate):
             self.wxVersion = ''
 
         #check for updates, if able
-        import pip
         pypi = xmlrpclib.ServerProxy('https://pypi.python.org/pypi')
-        available = pypi.package_releases('MAVProxy')
-        if not available:
+        try:
+            available = pypi.package_releases('MAVProxy')
+            if not available:
+                self.newversion = 'Error finding update'
+            else:
+                self.newversion = available[0]
+        except:
```

Comment:
Please try to avoid catching Exception like this; try to catch the relevant subset of exceptions.

Reply:
Fixed